_pages/about-us/offices/distributed.md

```diff
@@ -36,49 +36,49 @@ title: Distributed
 
 _Before you do anything, read our [best practices for making distributed teams work](https://18f.gsa.gov/2015/10/15/best-practices-for-distributed-teams/)._
 
-## <a id="how-do-i">How do I&hellip;</a>
+## How do I&hellip;
 
-### <a id="set-up-my-computer">Set up my computer?</a>
+### Set up my computer?
 
 TTS employees receive a MacBook and an iPhone from GSA, though you may not receive your work phone on your first day.
 
 On your first day at work you should configure things so that your two-factor authentication (2FA) codes &mdash; which you&rsquo;ll occasionally need to connect to GSA&rsquo;s virtual private network (VPN) &mdash; are delivered to your cell phone. If you don&rsquo;t receive your work cell phone on day one, you&rsquo;ll need to use your personal phone. **If you do not configure 2FA you will not be able to access your email from outside of federal buildings.**
 
 See the [Equipment page]({{site.baseurl}}/equipment/#phone) for more about how to set up 2FA so you can receive GSA 2FA codes on your cell phone.
 
-### <a id="otp">Get a one-time password?</a>
+### Get a one-time password?
```

Comment:
Some of these other H3s below here are also changing without manual control. I won't call them out individually.